packages/components/src/components/DetailsSearch.vue

```diff
@@ -191,6 +191,7 @@ export default {
 
   methods: {
     selectObject(type, object) {
+      this.$root.$emit('selectObjectInSidebar', type);
```

Comment:
FYI we're using `stateChange` elsewhere to pick up on selectedObject changes:
https://github.com/getappmap/vscode-appland/blob/master/web/src/appmapView.js#L87-L106

It won't tell you whether or not it was selected via the sidebar though.

Reply:
Yeah, similar to the above comment, I wanted a way to see if something was selected via the sidebar specifically. I'm also not sure that the `stateChange` event is working correctly since I can't see any `selected_object` events https://portal.azure.com#@d03f5316-f32a-4633-a0ec-dc62399e533b/blade/Microsoft_OperationsManagementSuite_Workspace/Logs.ReactView/resourceId/%2Fsubscriptions%2Fa4af4729-21ee-4e0a-953c-33cc9db5295c%2FresourceGroups%2Fvscode-extension/source/LogsBlade.AnalyticsShareLinkToQuery/q/H4sIAAAAAAAAA3MsKHAtS80rKeaqUSjPSC1KVfBLzE1VSM7PK0nMzCtWUCpOzUlNLklNic9PygIylKwB70eFojIAAAA%253D/timespan/P7D.